packages/__globalPreview/preview.html

```diff
@@ -14,11 +14,15 @@
       <div class="su-upper-content"></div>
       <div class="su-page-content">
         [component://output]
+        <script src="//ucomm-go.stanford.edu/js/forms2/js/forms2.min.js"></script>
+        <form class="marketo-sr-subscribe-form" id="mktoForm_1001"></form>
+        <script>MktoForms2.loadForm("//ucomm-go.stanford.edu", "660-TJC-984", 1001);</script>
         <div class="su-lower-content" id="component_130052"></div>
         <div class="su-lower-content"></div>
       </div>
       <div class="su-above-footer"></div>
     </main>
     [component://static-footer]
+      <script src="https://sug-web.matrix.squiz.cloud/__data/assets/js_file/0020/141455/marketo.js"></script>
```

Comment:
Reminder to myself: Remove this before merging

Reply:
I removed these before merging by the way 👍